modules/sdpolicy-am-billing.adoc

```diff
@@ -3,12 +3,41 @@
 // * osd_architecture/osd_policy/osd-service-definition.adoc
 :_content-type: CONCEPT
 [id="billing_{context}"]
-= Billing
-Each {product-title} cluster requires a minimum annual base cluster purchase and there are two billing options available for each cluster: Standard and Customer Cloud Subscription (CCS).
+= Billing options
 
-Standard {product-title} clusters are deployed in to their own cloud infrastructure accounts, each owned by Red Hat. Red Hat is responsible for this account, and cloud infrastructure costs are paid directly by Red Hat. The customer only pays the Red Hat subscription costs.
+Customers have the option to purchase annual subscriptions of {product-title} (OSD) or consume on-demand through cloud marketplaces. Customers can decide to bring their own cloud infrastructure account, referred to as Customer Cloud Subscription (CCS), or deploy in cloud provider accounts owned by Red Hat. The table below provides additional information regarding billing, as well as the corresponding supported deployment options.
+[cols="2a,3a,3a",options="header"]
+|===
 
-In the CCS model, the customer pays the cloud infrastructure provider directly for cloud costs and the cloud infrastructure account is part of a customer’s Organization, with specific access granted to Red Hat. In this model, the customer pays Red Hat for the CCS subscription and pays the cloud provider for the cloud costs. It is the customer's responsibility to pre-purchase or provide Reserved Instance (RI) compute instances to ensure lower cloud infrastructure costs.
+|OSD Subscription-type
+|Cloud infrastructure account
+|Billed through
+
+.2+|Annual fixed capacity subscriptions through Red Hat |Red Hat cloud account
```

Comment:
I have no advice for this part but for the next two [marketplace subscription type](https://64901--docspreview.netlify.app/openshift-dedicated/latest/osd_architecture/osd_policy/osd-service-definition). From the design document, there are two kinds of subscription-types: `Annual` and `On-demand`, either `google cloud marketplace` or `red hat marketplace` is one type of `On-demand`.  Could `google cloud marketplace` and `red hat marketplace` be at the same level of 'Annual'?

Besides, do not see `Customer’s own Google Cloud account`, there is `Customer’s own cloud account` for `google cloud marketplace` and `red hat marketplace`.

Reply:
Hello @yasun1, thanks for this feedback. I am going to bring in @svmrh to see if he can respond to this question. I believe the table is correct, but want to see if he has on opinion to this inquiry. Thanks!

Reply:
@yasun1 - can you please clarify, what do you mean by -

> Could google cloud marketplace and red hat marketplace be at the same level of 'Annual'?

I am little confused, because, Google Cloud Marketplace and Red Hat Marketplace cannot be at the same level as Annual.  This is because, they are NOT annual and purely on-demand/consumption-based. Please let me know if I interpreted your comment incorrectly, 

We decided to add three different rows to highlight the difference in infrastructure type and how many/who bills the customer.

Reply:
> I am little confused, because, Google Cloud Marketplace and Red Hat Marketplace cannot be at the same level as Annual. This is because, they are NOT annual and purely on-demand/consumption-based. Please let me know if I interpreted your comment incorrectly,
Agree with you. So I was thinking that in the table, the first level is `Annual` -> `On-demand`, and on-demand is separated into `Google Cloud Marketplace` and `Red Hat Marketplace`. 

Now, I rethink again, the current tab structure could be accepted too. Thanks.